Gemfile.lock

```diff
@@ -7,41 +7,73 @@ PATH
 GEM
   remote: https://rubygems.org/
   specs:
-    diff-lcs (1.1.3)
-    ffi (1.1.5)
-    guard (1.4.0)
-      listen (>= 0.4.2)
-      thor (>= 0.14.6)
-    guard-rspec (2.1.0)
-      guard (>= 1.1)
-      rspec (~> 2.11)
-    listen (0.5.3)
-    mock_redis (0.5.4)
-    rake (0.9.2.2)
-    rb-fsevent (0.9.2)
-    rb-inotify (0.8.8)
-      ffi (>= 0.5.0)
-    redis (3.2.0)
-    rspec (2.11.0)
-      rspec-core (~> 2.11.0)
-      rspec-expectations (~> 2.11.0)
-      rspec-mocks (~> 2.11.0)
-    rspec-core (2.11.1)
-    rspec-expectations (2.11.3)
-      diff-lcs (~> 1.1.3)
-    rspec-mocks (2.11.3)
-    thor (0.16.0)
-    timecop (0.5.2)
+    coderay (1.1.3)
+    diff-lcs (1.5.0)
+    ffi (1.15.5)
+    formatador (1.1.0)
+    guard (2.18.0)
+      formatador (>= 0.2.4)
+      listen (>= 2.7, < 4.0)
+      lumberjack (>= 1.0.12, < 2.0)
+      nenv (~> 0.1)
+      notiffany (~> 0.0)
+      pry (>= 0.13.0)
+      shellany (~> 0.0)
+      thor (>= 0.18.1)
+    guard-compat (1.2.1)
+    guard-rspec (4.7.3)
+      guard (~> 2.1)
+      guard-compat (~> 1.1)
+      rspec (>= 2.99.0, < 4.0)
+    listen (3.7.1)
+      rb-fsevent (~> 0.10, >= 0.10.3)
+      rb-inotify (~> 0.9, >= 0.9.10)
+    lumberjack (1.2.8)
+    method_source (1.0.0)
+    mock_redis (0.31.0)
+      ruby2_keywords
+    nenv (0.3.0)
+    notiffany (0.1.3)
+      nenv (~> 0.1)
+      shellany (~> 0.0)
+    pry (0.14.1)
+      coderay (~> 1.1)
+      method_source (~> 1.0)
+    rake (13.0.6)
+    rb-fsevent (0.11.1)
+    rb-inotify (0.10.1)
+      ffi (~> 1.0)
+    redis (4.6.0)
+    rspec (3.11.0)
```

Comment:
### [rspec](http://github.com/rspec)

**Major** version upgrade :chart_with_upwards_trend::exclamation: 2.11.0 → 3.11.0

[_~~change-log~~, [source-code](https://github.com/rspec/rspec-metagem), [gem-diff](https://my.diffend.io/gems/rspec/2.11.0/3.11.0)_]





<details>
<summary>Commits</summary>

A change of **94** commits. See the full changes on [the compare page](https://github.com/rspec/rspec-metagem/compare/v2.11.0...v3.11.0).

These are the first 10 commits:
- (b8e9f83) [bump version to 2.12](https://github.com/rspec/rspec-metagem/commit/b8e9f832cc56966ff48c47d5ec1f918c38986380)
- (3fdda41) [2.13.0 release](https://github.com/rspec/rspec-metagem/commit/3fdda41ff7dc5dbf4b0ea77ceaf11c9828cf2283)
- (be31e37) [Uses secure RubyGems URL](https://github.com/rspec/rspec-metagem/commit/be31e37cb8572774aac681c5be04e545f5799c77)
- (69994d0) [Merge pull request #2 from vfrride/patch-1](https://github.com/rspec/rspec-metagem/commit/69994d0e78876c873d3456b08d3a11030c3e0ac4)
- (4d279d8) [Release 2.14.0.rc1](https://github.com/rspec/rspec-metagem/commit/4d279d85e4487019d6c0195bcfb6ba1dbe23a23f)
- (047df66) [Release 2.14.0](https://github.com/rspec/rspec-metagem/commit/047df663cd932da4446b2574b2ac16e599735754)
- (a3daf2b) [This branch is 3.0.0.pre now](https://github.com/rspec/rspec-metagem/commit/a3daf2bd008d21e8d9e553946b3709d032ac5e84)
- (f10bedd) [Don't explicitly require mocks and expectations upfront.](https://github.com/rspec/rspec-metagem/commit/f10bedd498dff2961d4be9cce793fa152c4a0bbe)
- (90e3db8) [Update gem spec.](https://github.com/rspec/rspec-metagem/commit/90e3db8c90cd9423a9d89a32c9dee2c1444223e4)
- (07c4132) [Uses rspec-support from the local dev directory](https://github.com/rspec/rspec-metagem/commit/07c41323d79155445227bf42af2e4d4800cc0fcf)


</details>